go.mod

```diff
@@ -2,15 +2,42 @@ module github.com/hashicorp/terraform-provider-tfe
 
 go 1.16
 
+replace github.com/hashicorp/go-tfe => /Users/omarismail/work/go-tfe
```

Comment:
Once [this](https://github.com/hashicorp/go-tfe/pull/201) is merged, I'll remove this